internal/operators/mce/config.go

```diff
@@ -13,6 +36,34 @@ const (
 	SNOMinimumCpu    int64 = 8
 )
 
-type Config struct {
-	MceMinOpenshiftVersion string `envconfig:"MCE_MIN_OPENSHIFT_VERSION" default:"4.10.0"`
+func ParseMCEConfig(config Config) (*ParsedConfig, error) {
+	var parsedMatrix []OCPMCEReleaseMatrix
+	if err := json.Unmarshal([]byte(config.OcpMceReleaseMatrix), &parsedMatrix); err != nil {
+		return nil, fmt.Errorf("failed to unmarshal ocp mce release matrix: %w", err)
+	}
+
+	return &ParsedConfig{
```

Comment:
Why two structs? Just return the `OCPMCEReleaseMatrix`, no?

Reply:
asked a question about MceMinOpenshiftVersion

Reply:
Because if we are keeping `MceMinOpenshiftVersion` configurable it is necessary for marshaling